pywbem_mock/_wbemconnection_mock.py

```diff
@@ -0,0 +1,1571 @@
+#!/usr/bin/env python
+
+# (C) Copyright 2017 IBM Corp.
+# (C) Copyright 2017 Inova Development Inc.
+# All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Mock of WBEMConnection to enable tests of pywbem without a WBEM server.
+
+This class mocks all of the pywbem WBEMConnectionoperations that communicate
```

Comment:
Insert space in "WBEMConnectionoperations".

Reply:
Done

Reply:
DONE